strict-addons/es6/imports.js

```diff
@@ -0,0 +1,128 @@
+        'import/named': 'warn',
+
+        // ensure default import coupled with default export
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/default.md
+        'import/default': 'warn',
+
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/namespace.md
+        'import/namespace': 'error',
+
+        // disallow invalid exports, e.g. multiple defaults
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/export.md
+        'import/export': 'error',
+
+        // do not allow a default import name to match a named export
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default.md
+        'import/no-named-as-default': 'error',
+
+        // warn on accessing default export property names that are also named exports
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default-member.md
+        'import/no-named-as-default-member': 'error',
+
+        // Forbid the use of extraneous packages
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md
+        'import/no-extraneous-dependencies': ['error', {
+            devDependencies: [
+                '**/*.test.js', // ignore for tests
+                '**/webpack.config.js', // webpack config
+                '**/webpack.config.*.js', // webpack config
+                '**/gulpfile.js', // gulp config
+                '**/gulpfile.*.js' // gulp config
+            ]
+        }],
+
+        // Forbid mutable exports
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-mutable-exports.md
+        'import/no-mutable-exports': 'error',
+
+        // disallow require()
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-commonjs.md
+        'import/no-commonjs': 'off',
+
+        // disallow AMD require/define
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-amd.md
+        'import/no-amd': 'error',
+
+        // No Node.js builtin modules
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-nodejs-modules.md
+        'import/no-nodejs-modules': 'off',
+
+        // disallow non-import statements appearing before import statements
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md
+        'import/first': ['warn', 'absolute-first'],
+
+        // disallow duplicate imports
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md
+        'import/no-duplicates': 'error',
+
+        // Ensure consistent use of file extension within the import path
+        // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md
+        'import/extensions': ['warn', 'always', {

```

Comment:
good point. i think i'll switch the warns to errors and fix em up in payscale rc